.jitpack.yml

```diff
@@ -0,0 +1,5 @@
+jdk:
+  - openjdk8
+
+install:
+  - ./gradlew --no-daemon --stacktrace -x reporter-web-app:yarnBuild publishToMavenLocal
```

Comment:
Please add a comment explaining that the reporter-web-app task needs to be excluded because of issues with installing Yarn on Jitpack.

Reply:
Done.